src/StoreKeeper/WooCommerce/B2C/Imports/OrderImport.php

```diff
@@ -199,4 +210,22 @@ protected function getImportEntityName(): string
     {
         return __('orders', I18N::DOMAIN);
     }
+
+    /**
+     * @param $order
+     *
+     * @throws Exception
+     */
+    public static function fetchOrderStatusUrl(&$order, int $storekeeperId): void
+    {
+        $apiWrapper = StoreKeeperApi::getApiByAuthName();
+        $shopModule = $apiWrapper->getModule('ShopModule');
+        try {
+            $orderStatusPageUrl = $shopModule->getOrderStatusPageUrl($storekeeperId);
+            $order->update_meta_data(self::ORDER_PAGE_META_KEY, $orderStatusPageUrl);
+            $order->save();
+        } catch (GeneralException $generalException) {
+            LoggerFactory::create('order')->error($generalException->getMessage(), $generalException->getTrace());
+        }
+    }
```

Comment:
```suggestion
    public static function fetchOrderStatusUrl(&$order, int $storekeeperId): ?string
    {
        $apiWrapper = StoreKeeperApi::getApiByAuthName();
        $shopModule = $apiWrapper->getModule('ShopModule');
        $orderStatusPageUrl = null;
        try {
            $orderStatusPageUrl = $shopModule->getOrderStatusPageUrl($storekeeperId);
            $order->update_meta_data(self::ORDER_PAGE_META_KEY, $orderStatusPageUrl);
            $order->save();
        } catch (GeneralException $generalException) {
            LoggerFactory::create('order')->error($generalException->getMessage(), $generalException->getTrace());
        }
    }
    return $orderStatusPageUrl;
```

Maybe we can use it instead if it back from order if it fails?

Reply:
Changes implemented.